### PR TITLE
make things no selectable on mobile + enable touch zoom but disable r…

### PR DESCRIPTION
**Pull request by tonypls**
…otation

## Issue
https://linear.app/electricitymaps/issue/ELE-1621/deploy-mobile-apps
## Description
Preparing app builds and release, I'm sure there will be more but these are the first fixes I noticed running the android APK

### CODE REVIEW

File: web/src/features/map/Map.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ export default function MapPage(): ReactElement {
     if (!map || isLoading || isError) {
       return;
     }
-
+    map.touchZoomRotate.disableRotation();
```
Comment (madsnedergaard) [inline web/src/features/map/Map.tsx:99]: Did `touchZoomRotate={false}` not work?

File: web/src/index.css [hunk 2]
```diff
@@ -22,6 +22,7 @@
     @apply antialiased;
     @apply bg-white dark:bg-gray-800;
     @apply text-gray-900 dark:text-gray-200;
+    -webkit-tap-highlight-color: transparent;
```
Comment (madsnedergaard) [inline web/src/index.css:25]: Nice, is this the blue box that we thought was just Chrome dev tools? :D

**Review by madsnedergaard — APPROVED** [review]
Two minor questions, but LGTM

File: web/src/features/map/Map.tsx (hunk 1 above)
Comment (tonypls) [reply web/src/features/map/Map.tsx:99]: It disables both zoom and rotate with touch so you couldn't zoom 🧐

File: web/src/index.css (hunk 2 above)
Comment (tonypls) [reply web/src/index.css:25]: Yeah that's the one, I tested on Android and dev tools and looks good. I'll check iOS when I get a build done today

File: web/src/features/map/Map.tsx (hunk 1 above)
Comment (madsnedergaard) [reply web/src/features/map/Map.tsx:99]: Oooh okay, weird that it dodes both in one :)

**Pull request merged**